### PR TITLE
OSDOCS-2254: Post-GA STS cleanup

### PR DESCRIPTION
**Pull request by jonquilwilliams**
JIRA: https://issues.redhat.com/browse/OSDOCS-2254

Previews:

Creating cluster: https://deploy-preview-34534--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa-sts-creating-cluster.html#rosa-sts-creating-cluster_rosa-sts-getting-started

VPC: https://deploy-preview-34534--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa-sts-aws-prereqs.html#rosa-vpc_rosa-sts-aws-prerequisites

Security Requirements: https://deploy-preview-34534--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa-sts-aws-prereqs.html#rosa-security-requirements_rosa-sts-aws-prerequisites

Peer reviewer: Please merge to `dedicated-4` branch. No milestones and no CP. Thanks.

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Looks good to me!

**Comment by xueli181114** [comment]
Fag --operator-iam-roles is deprecated in our current latest errata release. It supports --operator-roles-prefix instead.
And
It supports create account-roles and operator-roles now.
Should we clarify it in the doc?
@vkareh

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 1]
```diff
@@ -31,8 +31,8 @@ See link:https://access.redhat.com/documentation/en-us/openshift_container_platf
...
 
-. Managed clusters require Telemetry to be enabled to allow Red Hat to react more quickly to problems and better support our customers, as well as better understand how product upgrades impact clusters.  
-See link:https://docs.openshift.com/container-platform/4.6/support/remote_health_monitoring/about-remote-health-monitoring.html[About remote health monitoring] for more information about how remote health monitoring data is used by Red Hat.
+. Managed clusters require Telemetry to be enabled to allow Red Hat to react more quickly to problems and better support our customers, as well as better understand how product upgrades impact clusters.
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/support/remote-health-monitoring-with-connected-clusters#about-remote-health-monitoring[About remote health monitoring] for more information about how remote health monitoring data is used by Red Hat.
```
Comment (xueli181114) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:35]: 4.8.2 is current default version on production env. Should the url be changed to https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html/support/remote-health-monitoring-with-connected-clusters?

File: modules/rosa-aws-provisioned.adoc [hunk 2]
```diff
@@ -57,16 +57,22 @@ Two buckets are required with a typical size of 2TB each.
 
 [id="rosa-vpc_{context}"]
 == VPC
-Customers should expect to see one VPC per cluster. Additionally, the VPC will need the following configurations:
+Customers should expect to see one VPC per cluster. Additionally, the VPC will need the following configurations.
 
-* *Subnets*: Two subnets for a cluster with a single availability zone, or six subnets for a cluster with multiple availability zones.
+=== Without AWS PrivateLink
 
+* *Subnets*: Two subnets for a cluster with a single availability zone, or six subnets for a cluster with multiple availability zones.
```
Comment (xueli181114) [inline modules/rosa-aws-provisioned.adoc:64]: There should be some clarification for `six subnets for a cluster with multiple availability zones.` that two subnets per zone

File: modules/rosa-sts-creating-cluster.adoc [hunk 3]
```diff
@@ -22,7 +22,7 @@ When creating your cluster, use the following default options to configure your
 
 . You can create a cluster by specifying the specifying the custom settings shown below or by using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
 +
-Creating a cluster can take up to 40 minutes.
+Creating a cluster can take up to 40 minutes. During this waiting period, you can configure identity providers that determine how users log in to access the cluster. For more information, see Next Steps below.
```
Comment (xueli181114) [inline modules/rosa-sts-creating-cluster.adoc:25]: The below steps are for how to configure oidc-providers and how to create operator-roles on AWS. Those steps are not used for user to configure `you can configure identity providers that determine how users log in to access the cluster` . I think this sentence should be moved to the last step 5 in file https://deploy-preview-34534--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa-sts-creating-cluster.html#rosa-sts-creating-cluster_rosa-sts-getting-started
And
We should better to clarify that cluster will keep pending until the oidc-provider created in step 4

**Pull request closed without merging**